db/seeds.rb

```diff
@@ -9,10 +9,18 @@
 # bin/rails c    OR   rake db:reset AND bin/rails db:seed
 # rake db:drop db:create db:migrate db:seed
 
-6.times do |prize|
-  Prize.create(name: "Tshirt #{prize + 1}", percentage: 10, image: "assets/svgs/prizes/t-shirt.svg", prize_type: 'Merch', description: 'This is a merch item', identifier: 'merch')
-  Prize.create!(name: "NFT #{prize + 1}", percentage: 10, image: "assets/svgs/prizes/nft.svg", prize_type: 'NFT', description: 'This is a NFT item', identifier: 'NFT')
-end
+Prize.create(name: "T-shirt", percentage: 15, image: "assets/svgs/svgs/prizes/t-shirt.svg", prize_type: 'Merch', description: 'This is a Merch item', identifier: 'merch')
+Prize.create(name: "Pen", percentage: 15, image: "assets/svgs/prizes/pen.svg", prize_type: 'Merch', description: 'This is a Merch item', identifier: 'merch')
+Prize.create(name: "Hoodie", percentage: 15, image: "assets/svgs/prizes/hoodie.svg", prize_type: 'Merch', description: 'This is a Merch item', identifier: 'merch')
+Prize.create(name: "Keychain", percentage: 15, image: "assets/svgs/prizes/keychain.svg", prize_type: 'Merch', description: 'This is a Merch item', identifier: 'merch')
+Prize.create(name: "Tote Bag", percentage: 15, image: "assets/svgs/prizes/tote-bag.svg", prize_type: 'Merch', description: 'This is a Merch item', identifier: 'merch')
+Prize.create(name: "Hat", percentage: 15, image: "assets/svgs/prizes/hat.svg", prize_type: 'Merch', description: 'This is a Merch item', identifier: 'merch')
+Prize.create(name: "CryptoPunks NFT", percentage: 2, image: "assets/svgs/prizes/nft.svg", prize_type: 'NFT', description: 'This is a NFT item', identifier: 'NFT')
+Prize.create(name: "Moonbirds NFT", percentage: 2, image: "assets/svgs/prizes/nft.svg", prize_type: 'NFT', description: 'This is a NFT item', identifier: 'NFT')
+Prize.create(name: "Pudgy Penguins NFT", percentage: 2, image: "assets/svgs/prizes/nft.svg", prize_type: 'NFT', description: 'This is a NFT item', identifier: 'NFT')
+Prize.create(name: "Cool Cat NFT", percentage: 2, image: "assets/svgs/prizes/nft.svg", prize_type: 'NFT', description: 'This is a NFT item', identifier: 'NFT')
+Prize.create(name: "Mutant Ape NFT", percentage: 1, image: "assets/svgs/prizes/nft.svg", prize_type: 'NFT', description: 'This is a NFT item', identifier: 'NFT')
+Prize.create(name: "Bored Ape NFT", percentage: 1, image: "assets/svgs/prizes/nft.svg", prize_type: 'NFT', description: 'This is a NFT item', identifier: 'NFT')
```

Comment:
I know it's just the seeds, but it still feels like an excellent opportunity to practice writing some arrays and objects and looping over them, no?

Reply:
Ok sure, I'll fix it